pyvelox/test/test_vector.py

```diff
@@ -254,3 +255,35 @@ def test_append(self):
 
         with self.assertRaises(TypeError):
             ints2.append(strs2)
+
+    def test_export_to_arrow(self):
```

Comment:
Thanks this is great, can we add export / import tests for all the types we support currently though ?

Reply:
It's certainly good to add some more tests, but just to mention that the actual conversion itself (the C++ code) is also already tested at https://github.com/facebookincubator/velox/blob/main/velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

Reply:
Thanks @jorisvandenbossche , nevertheless would be good to add tests for atleast the primitive types to make sure there are no inadvertent casts across types , since its going via Cpp to Python etc. Also PyVelox doesnt support all the types thats Velox supports yet :) .

Reply:
I have added the test cases for integers, floats, and strings. Should I add some other test cases as well?

Reply:
Also, while testing multiple test cases, I noticed, that the memory pool instance was not handled correctly earlier. So, I had to move the Instance struct to a separate header file, does this approach look good, or can there be a better way?